tests/service/Common.fs

```diff
@@ -237,25 +230,25 @@ let attribsOfSymbol (s:FSharpSymbol) =
 //            if v.LiteralValue.IsSome then yield "literal"
         | _ -> () ]
 
-let rec allSymbolsInEntities compGen (entities: IList<FSharpEntity>) = 
```

Comment:
This PR has a large number of changes removing whitespace from end-of-line.  Please undo all of these and make sure the diff is minimal, thanks